README.rst

```diff
@@ -3,5 +3,15 @@ ZeroZero
 
 A documentation repos to describe the common philosophy behind the educational *Zero projects.
 
-
 Not a lot here, but this will change soon enough.
+
+*Zero Projects
```

Comment:
Yeah, I only noticed it when looking at the github rendered version (https://github.com/bennuttall/zerozero/blob/patch-1/README.rst ), wasn't sure if they have their own ReStructuredText flavour or it was a typo. I'm pretty sure I've seen other documentation in rst format not rendered properly in GitHub but producing the static html documentation as expected, so once again never found out if that's due a "github flavour" or "static-builder flavour".